src/iops.zig

```diff
@@ -1,53 +1,56 @@
 const std = @import("std");
+const stdx = @import("./stdx.zig");
 const assert = std.debug.assert;
 
 /// Take a u8 to limit to 256 items max (2^8 = 256)
 pub fn IOPSType(comptime T: type, comptime size: u8) type {
-    const Map = std.StaticBitSet(size);
+    const Map = stdx.BitSetType(size);
+
     return struct {
         const IOPS = @This();
 
         items: [size]T = undefined,
-        /// 1 bits are free items.
-        free: Map = Map.initFull(),
+        busy: Map = .{},
 
         pub fn acquire(self: *IOPS) ?*T {
-            const i = self.free.findFirstSet() orelse return null;
-            self.free.unset(i);
+            const i = self.busy.first_unset() orelse return null;
+            self.busy.set(i);
             return &self.items[i];
         }
 
         pub fn release(self: *IOPS, item: *T) void {
             item.* = undefined;
             const i = self.index(item);
-            assert(!self.free.isSet(i));
-            self.free.set(i);
+            assert(self.busy.is_set(i));
+            self.busy.unset(i);
         }
 
         pub fn index(self: *IOPS, item: *T) usize {
-            const i = @divExact((@intFromPtr(item) - @intFromPtr(&self.items)), @sizeOf(T));
+            const i = @divExact(
+                (@intFromPtr(item) - @intFromPtr(&self.items)),
+                @sizeOf(T),
+            );
```

Comment:
Unnecessary line break, still under 100 columns.

Reply:
Yeah, I just find it poorly readable without a line break: it is a fraction, so we might as well make it look like a fraction!